step.yml

```diff
@@ -0,0 +1,60 @@
+title: |-
+  Android Lint
+summary: |
+  Runs lint gradle task on your project.
+description: |
+  Runs lint gradle task on your project.
+website: https://github.com/bitrise-steplib/bitrise-step-android-lint
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-android-lint
+support_url: https://github.com/bitrise-steplib/bitrise-step-android-lint/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+project_type_tags:
+  - android
+  - react-native
+
+type_tags:
+  - test
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-android-lint
+
+inputs:
+  - project_location: $BITRISE_SOURCE_DIR
+    opts:
+      title: Project Location
+      summary: A project in Android Studio contains everything that defines your workspace for an app (gradlew, settings.gradle), from source code and assets, to test code and build configurations.
```

Comment:
The root directory of your android project, e.g.: where your root build gradle file exist.